cmd/schemagen/schemagen_test.go

```diff
@@ -30,7 +30,7 @@ func TestSchemagen(t *testing.T) {
 
 	b := runSchemagen(t, "foobar")
 
-	const goldenFile = "testdata/models.go.txt"
```

Comment:
What's the reason for a file name change?

Reply:
It has changed since I want to test generated code for real